src/CSET/recipes/generic_surface_histogram_series.yaml

```
@@ -0,0 +1,36 @@
+category: Quick Look
+title: $VARNAME Surface Level Histogram Plot
+description: |
+  Extracts and plots the probability density of surface $VARNAME. It uses
+  matplotlib.pyplot.hist to plot the probability density so that the area under
+  the histogram integrates to 1. stacked is set to True so the sum of the
+  histograms is normalized to 1.
```

```suggestion
  Extracts and plots the probability density of surface `$VARNAME`. It uses
  [`plt.hist`](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.hist.html)
  to plot the probability density so that the area under the histogram
  integrates to 1. stacked is set to True so the sum of the histograms is
  normalized to 1.
```
These descriptions are Markdown, so we can put a proper link in.